borik.go

```diff
@@ -6,12 +6,18 @@ import (
 	"os/signal"
 	"syscall"
 
+	"github.com/joho/godotenv"
 	"github.com/rs/zerolog/log"
 	"github.com/saturn-sh/borik/bot"
 	"gopkg.in/gographics/imagick.v2/imagick"
 )
 
 func main() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Info().Msg("Failed to load .env file")
```

Comment:
```suggestion
		log.Info().Err(err).Msg("Failed to load .env file")
```

Reply:
zerolog isn't initialized yet, so it spits out json blob, using Printf instead